WTWSMS/localisation/WtWSMS_religions.csv

```diff
@@ -148,7 +148,7 @@ d_sethian_holy_order_adj;Snake Knight;Chevalier serpent;;;;;;;;;x
 #k_nicene;The Nicene Church;L'�glise Nic�en;;;;;;;;;;x
 #k_nicene_adj;Nicene;Nic�en;;;;;;;;;;x
 nicene;Chalcedonian;chalc�donienne;;;;;;;;;;x
-nicene_desc;Born out of the Council of Chalcedon in 451, the Chalcedonian Christianity is the central church of the Pentarchy, and the main faith of the two empires.;;;;;;;;;;;x
+nicene_DESC;Born out of the Council of Chalcedon in 451, the Chalcedonian Christianity is the central church of the Pentarchy, and the main faith of the two empires.;N�e lors du concile de Chalc�doine en 451, le christianisme chalc�donienne est l'�glise centrale de la Pentarchie, et la foi principale des deux empires.;;;;;;;;;;x
```

Comment:
It should be _chalcédonien_ in masculine, not _chalcédonienne_ in feminine.

Reply:
Still simpler than in my language...
Fixed ✔️

Reply:
Thanks, seems like there is an older mistake too, _Née_ should be _Né_.

Reply:
Done, also fixed in the previously used line.

Reply:
I don't want to nitpick with French grammar, but while you are at it nicene should also be _chalcédonien_ as well as k_nicene_adj. (k_nicene should remain _L'Église chalcédonienne_) That will be the last comment I make, I promise. 🙄

Reply:
Should be corrected now ✔️ 
TIL there are no grammatical cases in French
![image](https://user-images.githubusercontent.com/29546927/71845077-d184fc00-30c7-11ea-9eb9-4ed29b40c6c7.png)
I don't get why Hungarians would want 18.

Reply:
We are fortunate in that regard, yes. 😄 That being said silent consonants and such complicate the spelling enough for French to be a challenge anyway. :wink: